httpbeat.template-es2x.json

```diff
@@ -92,7 +92,7 @@
             },
             "headers": {
               "properties": {},
-              "type": "replace"
```

Comment:
@christiangalsterer can you tell me where this `replace` is coming from. I got an error when importing this in es 6 (the es6 template). So I thought it should be `object`, but now the CI fails because this type is unknown!?